README.md

```diff
@@ -1,40 +1,378 @@
+					"self": "http://example.com/articles/01FTZBPQ1EY5P5N3QW4ZHHFCM3/relationships/comments",
+					"related": "http://example.com/articles/01FTZBPQ1EY5P5N3QW4ZHHFCM3/comments"
+				},
+				"data": [{
+						"type": "comments",
+						"id": "01FTZBQRGVWX1AZG19NN0FMQZR"
+					},
+					{
+						"type": "comments",
+						"id": "01FTZBNTAB25AM278R6G3YRCKT"
+					}
+				]
+			}
+		},
+		"links": {
+			"self": "http://example.com/articles/01FTZBPQ1EY5P5N3QW4ZHHFCM3"
+		}
+	}]
+}
+```
+
+You would create this model:
 
 ```php
 <?php
 
 declare(strict_types=1);
 
+namespace App;
+
+use App\Article;
+use App\Comment;
+use Undabot\SymfonyJsonApi\Model\ApiModel;
+use Undabot\SymfonyJsonApi\Model\Resource\Annotation\Attribute;
+use Undabot\SymfonyJsonApi\Model\Resource\Annotation\ToMany;
+use Undabot\SymfonyJsonApi\Model\Resource\Annotation\ToOne;
+use Undabot\SymfonyJsonApi\Service\Resource\Validation\Constraint\ResourceType;
+
+/** @ResourceType(type="articles") */
+final class ArticleReadModel implements ApiModel
+{
+    public function __construct(
+        public readonly string $id,
+        /** @Attribute */
+        public readonly string $title,
+        /** @ToOne(name="author", type="authors") */
+        public readonly string $authorId,
+        /**
+         * @var array<int,string>
+         * @ToMany(name="comments", type="comments")
+         */
+        public readonly array $commentIds,
+    ) {
+    }
+
+    public static function fromSomething(Article $article): self
+    {
+        return new self(
+            (string) $article->id(),
+            (string) $article->title(),
+            (string) $article->author()->id(),
+            $article->comments()->map(static function (Comment $comment): string {
+                return (string) $comment->id();
+            })->toArray(),
+        );
+    }
+}
+```
+
+Same as in create and/or update model, read model consists of properties with annotations. Each property has annotation that states whether is it a relation or attribute. 
+
+Another part of this class is the static method `fromSomething` used when creating the read model. As previously mentioned, we often use `fromEntity` naming. In addition, we often use other namings for this method, such as `fromValueObject` or `fromAggregate`.
+The data flow that we usually use is similar to this:
+1. We fetch data from the database as an entity and work with that entity through our application.
+2. We use the entity until the moment in which we need to return the response.
+3. At that moment, we pass the entity to this library to create a proper read model.
+
+_In this example, we assume that there is a `title` method on the article entity which have `__toString` method, same as id. Also, we assume that this is Doctrine entity which have `author` method which is `Author` entity and `comments` method which is `Doctrine Collection` of `Comment` objects. But this can be any object from which you can construct a given read model._
```

Comment:
@KondukterCRO na ovom dijelu sam se izgubila, trebaš mi reći što si htio napisati